suites/apis/dataclientTest.js

```diff
@@ -0,0 +1,253 @@
+/* eslint-disable no-undef */
+/* eslint-disable consistent-return */
+/* eslint-disable max-len */
+
+// what does this test do:
+// 1. downloads the latest gen3-client version and unzips in the working directory
+// 2. creates the api key and stores them in the creds.json file - filename 'credsFile'
+// 3. using the creds.json, configure the profile
+// 4. upload the file - filename 'fileToBeUploaded'
+// 5. check the file is uploaded in indexd
+// 6. download the file
+
+// this test will run in nightly-builds and gen3-qa PRs
+
+Feature('Gen3-Client CLI testing @requires-fence @requires-indexd');
+
+const { expect } = require('chai');
+const fs = require('fs');
+const { execSync } = require('child_process');
+const axios = require('axios');
+const AdmZip = require('adm-zip');
+const users = require('../../utils/user');
+// const os = require('os');
+
+const I = actor();
+
+const latestGen3client = 'https://github.com/uc-cdis/cdis-data-client/releases/latest/download/dataclient_linux.zip';
```

Comment:
not sure I get the idea of this CI: looks like it is download the binary from the latest release of gen3-client instead of from the current branch/tag? So it mean even with this tests in CI, we cannot catch regressions in gen3-client before commiting it into master and make a release?